arangod/Cluster/ClusterInfo.cpp

```diff
@@ -5726,6 +5726,30 @@ std::shared_ptr<std::vector<ServerID> const> ClusterInfo::getResponsibleServer(
   return std::make_shared<std::vector<ServerID>>();
 }
 
+ClusterInfo::ShardLeadership ClusterInfo::getShardLeadership(
+    ServerID server, ShardID shard) const {
+  if (!_currentProt.isValid) {
+    return ShardLeadership::kUnclear;
+  }
+  READ_LOCKER(readLocker, _currentProt.lock);
+  auto it = _shardIds.find(shard);
+
+  if (it != _shardIds.end()) {
+    auto const& serverList = (*it).second;
+    if (!serverList || serverList->empty()) {
+      return ShardLeadership::kUnclear;
+    }
+    if (!(*serverList)[0].empty() && (*serverList)[0][0] == '_') {
+      // This is a temporary situation in which the leader has already
```

Comment:
the comment seems to be cut off. The comment appears in other places in this file, and there the comment ends with `in which the leader has already resigned.`.